docs/integrations/app-integrations/microsoft-teams.mdx

```diff
@@ -3,189 +3,105 @@ title: Microsoft Teams
+* `client_secret`: The client secret of the registered Microsoft Entra ID application.
 
 <Note>
-The chat bot will assume the identity of the user who signed in with their Microsoft account when the database was created.
+Microsoft Entra ID was previously known as Azure Active Directory (Azure AD).
 </Note>
 
-### Step 2: Create an [Agent](https://docs.mindsdb.com/agents/agent)
-
-An agent is created by combining a conversational model, like [LlamaIndex](https://docs.mindsdb.com/integrations/ai-engines/llamaindex), with a set of Skills.
-
-Here, we will create an agent without any skills, however, it is possilbe to create agents with skills such as the ability to answer questions from a knowledge base.
-
-#### Step 2.1: Create a Conversational Model
-
-Create a conversational model using either the [`LangChain`](https://docs.mindsdb.com/integrations/ai-engines/langchain) or [`LlamaIndex`](https://docs.mindsdb.com/integrations/ai-engines/llamaindex) integrations. Given below is an example of a conversational model created using the `LlamaIndex` integration:
-
-```sql
-CREATE ML_ENGINE llama_index_engine
-FROM llama_index
-USING openai_api_key='your-openai-api-key';
-```
-
-```sql
-CREATE MODEL llama_index_convo_model
-PREDICT answer
-USING
-  engine = 'llama_index_engine',
-  mode = 'conversational',
-  prompt = 'answer users questions as a helpful assistant',
-  user_column = 'question',
-  assistant_column = 'answer';
-```
-
-#### Step 2.2: Create an Agent using the Conversational Model
-
-Let's create an agent using the conversational model created above:
-
-```sql
-CREATE AGENT convo_agent
-USING
-    model = 'llama_index_convo_model',
-```
-
-### Step 3: Create a [Chatbot](https://docs.mindsdb.com/agents/chatbot)
-
-Finally, create a chatbot using the agent and the Microsoft Teams data source created above:
+## Usage
 
-```sql
-CREATE CHATBOT teams_chatbot
-USING
-    database = 'teams_datasource',
-    agent = 'convo_agent';
-```
+This integration can only be used to create chatbots for Microsoft Teams via the [`CREATE CHATBOT`](/agents/chatbot) syntax. Currently, *it cannot be used as a data source for other workloads*.
+
+Follow the instructions given below to set up the Microsoft Teams app that will act as the chatbot:
+
+  1. Follow [this link](https://dev.botframework.com/bots/new) to the Microsoft Bot Framework portal and sign in with your Microsoft account.
+  2. Fill out the *Display name*, *Bot handle*, and, optionally, the *Long description*, but leave the *Messaging endpoint* field empty for now.
+  3. Set the *App type* to be 'Multi Tenant' and click on *Create Microsoft App ID and password*. This will open a new tab with the Azure portal.
+  4. Click on *New registration* and fill out the *Name* and select the 'Accounts in any organizational directory (Any Azure AD directory - Multitenant)' option under *Supported account types*, and click on *Register*. Record the *Application (client) ID* for later use.
```

Comment:
Let's use the backticks to enclose values like \`Accounts in any organizational directory (Any Azure AD directory - Multitenant)\`